common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -198,71 +204,128 @@ def _build_fixture(self, unit_state):
             subsection if unit_state.publish_state == self.PublishState.NEVER_PUBLISHED
             else subsection.add_children(
                 XBlockFixtureDesc('vertical', name, metadata={
-                    'visible_to_staff_only': True if unit_state.is_locked else None
+                    'visible_to_staff_only': True if unit_state.is_locked else None,
+                    'group_access': {50: [1]} if unit_state.is_restricted else {},
```

Comment:
I think we have some constants in other tests for these values (see test_studio_container.py).

I don't see that there is test coverage for the dialog itself (either bok choy or sufficient unit test coverage). Can you explain which portions of the code change in this PR are covered by tests? We will want to ensure we have test coverage for using the dialog to control access.